buildSrc/src/main/kotlin/TargetsConfig.kt

```diff
@@ -52,31 +53,47 @@ fun Project.configureTargets() {
 
         if (hasPosix || hasLinux || hasDarwin || hasWindows) extra.set("hasNative", true)
 
-        sourceSets {
-            if (hasJsAndWasmShared) {
-                val commonMain by getting {}
-                val jsAndWasmSharedMain by creating {
-                    dependsOn(commonMain)
-                }
-                val commonTest by getting {}
-                val jsAndWasmSharedTest by creating {
-                    dependsOn(commonTest)
+        @OptIn(ExperimentalKotlinGradlePluginApi::class)
+        val ktorTargetsHierarchy = KotlinHierarchyTemplate {
```

Comment:
just curious: is there a reason to extract this into a separate variable instead of just calling `applyHierarchyTemplate {...}`?

Reply:
there is no reason. so it can be merged into one call.